### ISSUE TITLE
Styling axes - parameters not doing anything

### ISSUE THREAD

**Issue by FedeClaudi** [marcomusy/vtkplotter#146]
Hi Marco, 

I am trying to style the axes of a `Plotter`  by passing a bunch of parameters
with a dictionary. This is the dictionary:
```
    _default_axes_params = dict(
        xyGrid = True,
        xyPlaneColor = 'k',
        xyGridColor = 'red',
        xyAlpha = 1,
        axesLineWidth = 10,
        gridLineWidth = 50, 
    )
```
which I pass to the `axes` argument of `Plotter`. I got the parameters names form the `Plotter` docstring: https://github.com/marcomusy/vtkplotter/blob/e09cb33c2d4071ac39af4e140be68debdf1a0635/vtkplotter/plotter.py#L396

It seems that some parameters are not doing anything... 
`axesLineWidth` works, but all the others don't seem to be doing anything. 

A few more things I've tried:
passing  `_default_axes_params` to `show` and to `plotter.show` . I also tried using `show` with `newPlotter=True` but nothing changed. 

I can see the axes and change the line thickness, but the planes between axes never appear 
<img width="1680" alt="Screenshot 2020-05-12 at 20 43 07" src="https://user-images.githubusercontent.com/17436313/81738193-3b0b8180-9491-11ea-9b14-4fedf4c1c1ff.png">

**Comment by marcomusy** [comment]
Yes! this is actually a known problem ... there seems to be some weird incompatibility btw the depthpeeling algorithm and low transparency objects which belong to a `vtkAssembly`, I never managed to understand what's wrong with it, I can only that hope the problem will be fixed in VTK9.
